CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Enable drag-and-drop rows via `@dnd-kit` package.
```

Comment:
```suggestion
- option `dragAndDropOptions` to enable drag-and-drop rows via `@dnd-kit` package.
```

Reply:
just to directly mention how to enable it :)

Reply:
Thanks, done :-)